packages/components/src/FormField/FormField.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-statements */
```

Comment:
Can we avoid having this?

Reply:
no :( not without some refactors I don't think. the `handleKeyDown` method causes it.

Reply:
Could you add a comment about it perhaps then? Also, perhaps use a disable next line version instead of the whole file?